package.json

```diff
@@ -93,65 +100,28 @@
 		"npm": "^7.0.0"
```

Comment:
Can we just remove it?
It not affects the host's npm.

Reply:
No, this is what we requires at Nextcloud for now.
Min npm7 and node14. We move alltogether when npm7 got out and communicated about it already